content/api-reference/graphql-console.md

```diff
@@ -0,0 +1,86 @@
+
+1. Use the GraphQL Explorer tree to create a request.
+
+   - Use the selections under the `where:` argument to filter by fields that equal, contain, start with, or end with a specified value, or are included in an array of values.
+
+   - Use the selections under the `options:` argument to retrieve a specified count of objects or to sort the data returned. If you select multiple sort fields, the order in which you select them determines the sort order.
+
+   - For fields that can be arrays, select **+ Add** below them to add spaces for additional values.
+
+   - For upsert or delete operations, use the selections under `input:` to identify the object to update or delete.
+
+   - Select the fields to include in the response.
+
+   - Select **Reset** to clear all selections.
+
+   - Select ![copy](../_icons/branded/content-copy.svg) in the `Request` or `Variables` pane to copy the contents of that pane to the clipboard.
+
+   - Select ![GraphQL Explorer Pane](../_icons/branded/sitemap.svg) to hide or show the `GraphQL Explorer` pane.
+
+   - Select ![Response pane](../_icons/branded/response.svg) to hide or show the `Response` pane.
+
+   **Note:** The `GraphQL Explorer` and `Request` panes work in both directions. The request updates as you make selections in the GraphQL Explorer tree, and editing the text directly in the request or pasting a complete request updates the GraphQL Explorer tree. When editing the request directly, select ![Format](../_icons/default/auto-fix.svg) to automatically format the request.
+
+1. Select ![return](../_icons/branded/arrow-left-circle-outline.svg) to return to the top level and repeat the process with other objects, as needed.
+
+1. When your query is complete, select **Send Request** to run the query.
+
+   The results of your query appear in the `Response` pane.
+
+1. When you request a certain amount of fields, such as `count: 1`, and there are more fields available to display, a continuation token appears in the response. Select `Resend with Continuation` to view the next item.
+
+## Variables
+
+Rather than entering specific values for your query, you can use variables.
+
+1. Select the ![set variable](../_icons/custom/currency-usd-off.svg) icon that appears next to an argument when you hover over it.
+
+1. Enter a variable name in the box that appears.
+
+   A variable declaration is added to the `Request` pane and the new variable is added to the `Variables` pane in JSON format. The variable is assigned a value of `null` by default.
+
+1. Edit the value in the `Variables` pane to assign a value to the variable.
+
+   You can also make an array a variable and include multiple values, such as `[98, 99, ...]`.
+
+If you manually edit the variable name within the `Request` pane, select ![Format](../_icons/default/auto-fix.svg) to automatically update the variable name in the header and `Variables` pane.
```

Comment:
Just to confirm that there's no "ok" button after entering the variable or anything like that.